warnings from Eth

### Hardware:
Board:							ESP32-C3 devkit
Core Installation version:		master 22/10/2021
IDE name:						Arduino IDE
Computer OS:                    Ubuntu

### Description:
Warnings from Ethernet components in WiFi.
This does not prevent compilation.  I don't know if it prevents Eth from working, and don't have the stuff to test.  Just wanted to get eyes on this before v2.0.1 goes out the door.

### Sketch:  (leave the backquotes for [code formatting](https://help.github.com/articles/creating-and-highlighting-code-blocks/))
```cpp

#include <Arduino.h>
#include <WiFi.h>

void setup() {
}

void loop() {
}
```

### Debug Messages:
```
/home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/ETH.cpp: In member function 'bool ETHClass::begin(uint8_t, int, int, int, eth_phy_type_t, eth_clock_mode_t)':
/home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/ETH.cpp:240:46: warning: 'esp_err_t esp_eth_set_default_handlers(void*)' is deprecated [-Wdeprecated-declarations]
     if(esp_eth_set_default_handlers(eth_netif) != ESP_OK){
                                              ^
In file included from /home/larryb/Arduino/hardware/espressif/esp32/tools/sdk/esp32c3/include/esp_netif/include/esp_netif.h:26,
                 from /home/larryb/Arduino/hardware/espressif/esp32/tools/sdk/esp32c3/include/esp_event/include/esp_event_legacy.h:22,
                 from /home/larryb/Arduino/hardware/espressif/esp32/tools/sdk/esp32c3/include/esp_event/include/esp_event.h:28,
                 from /home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/WiFiGeneric.h:27,
                 from /home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/WiFiSTA.h:28,
                 from /home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/WiFi.h:32,
                 from /home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/ETH.h:24,
                 from /home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/ETH.cpp:21:
/home/larryb/Arduino/hardware/espressif/esp32/tools/sdk/esp32c3/include/esp_eth/include/esp_eth_netif_glue.h:59:11: note: declared here
 esp_err_t esp_eth_set_default_handlers(void *esp_netif)  __attribute__ ((deprecated));
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/ETH.cpp:240:46: warning: 'esp_err_t esp_eth_set_default_handlers(void*)' is deprecated [-Wdeprecated-declarations]
     if(esp_eth_set_default_handlers(eth_netif) != ESP_OK){
                                              ^
In file included from /home/larryb/Arduino/hardware/espressif/esp32/tools/sdk/esp32c3/include/esp_netif/include/esp_netif.h:26,
                 from /home/larryb/Arduino/hardware/espressif/esp32/tools/sdk/esp32c3/include/esp_event/include/esp_event_legacy.h:22,
                 from /home/larryb/Arduino/hardware/espressif/esp32/tools/sdk/esp32c3/include/esp_event/include/esp_event.h:28,
                 from /home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/WiFiGeneric.h:27,
                 from /home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/WiFiSTA.h:28,
                 from /home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/WiFi.h:32,
                 from /home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/ETH.h:24,
                 from /home/larryb/Arduino/hardware/espressif/esp32/libraries/WiFi/src/ETH.cpp:21:
/home/larryb/Arduino/hardware/espressif/esp32/tools/sdk/esp32c3/include/esp_eth/include/esp_eth_netif_glue.h:59:11: note: declared here
 esp_err_t esp_eth_set_default_handlers(void *esp_netif)  __attribute__ ((deprecated));
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/libraries/WiFi/src/ETH.cpp
+++ b/libraries/WiFi/src/ETH.cpp
@@ -237,12 +237,6 @@ bool ETHClass::begin(uint8_t phy_addr, int power, int mdc, int mdio, eth_phy_typ
     esp_netif_config_t cfg = ESP_NETIF_DEFAULT_ETH();
     esp_netif_t *eth_netif = esp_netif_new(&cfg);
 
-    if(esp_eth_set_default_handlers(eth_netif) != ESP_OK){
-        log_e("esp_eth_set_default_handlers failed");
-        return false;
-    }
-    
-    
     esp_eth_mac_t *eth_mac = NULL;
 #if CONFIG_ETH_SPI_ETHERNET_DM9051
     if(type == ETH_PHY_DM9051){

